x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -297,7 +300,15 @@ public void testCrossClusterQuery() throws Exception {
                     | LIMIT 2
                     | KEEP emp_id, department"""));
                 assertOK(response);
-                Map<String, Object> values = entityAsMap(response);
+                Map<String, Object> responseAsMap = entityAsMap(response);
+                List<?> columns = (List<?>) responseAsMap.get("columns");
+                List<?> values = (List<?>) responseAsMap.get("values");
+                assertEquals(2, columns.size());
+                assertEquals(2, values.size());
+                List<String> flatList = values.stream()
+                    .flatMap(innerList -> innerList instanceof List ? ((List<String>) innerList).stream() : Stream.empty())
+                    .collect(Collectors.toList());
+                assertThat(flatList, containsInAnyOrder("1", "3", "engineering", "sales"));
             }
             {
                 Response response = performRequestWithRemoteSearchUser(esqlRequest("""
```

Comment:
nit: can we also set and randomize `ccs_minimize_roundtrips` parameter

Reply:
We don't have `ccs_minimize_roundtrips` in ESQL